### PR TITLE
Update README to link to zenodo repo with checkpoint

### PR DESCRIPTION
**Pull request by oliverwm1**
Add a link to a zenodo data repository which contains model checkpoint and a 1-year subsample of the validation data.

### CODE REVIEW

File: examples/config-inference.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ logging:
   entity: your_wandb_entity
 validation_data:
   batch_size: 1
-  data_path: /path/to/validation/ic_0011
+  data_path: /path/to/data/directory
```
Comment (gideonite) [inline examples/config-inference.yaml]: question: Do we want to document what we expect to be present in this directory? e.g. netcdf files named in a particular way.

**Review by gideonite — COMMENTED** [review]
Looks like a big improvement. One question about the data_path

File: examples/config-inference.yaml (hunk 1 above)
Comment (oliverwm1) [reply examples/config-inference.yaml]: I can add a short comment, but mostly I expect users to infer this from the example data we provide.

**Review by gideonite — APPROVED** [review]
LGTM!

**Pull request merged**